src/pytest_plugins/consume_via_engine_api/consume_via_engine_api.py

```diff
@@ -0,0 +1,5 @@
+"""
```

Comment:
Just a thought that these tests are going to require the hive version of the fixtures.

Reply:
We should rebrand this to "engine" version of the fixtures :wink: 

In due course, we should probably fix the naming of `fill`'s `--enable-hive` flag. Let's decide how we package our fixtures first though.

Reply:
cc @spencer-tb

Reply:
I still feel like we should deprecate `--enable-hive`, and remove the hive fixtures completely. As now we can extract and process all this information directly from the default fixtures within the `consume_via_engine_api.py` simulator. 

I'll make 2 seperate PRs and we can see what we prefer - more discussion on this would be a good idea I think!

Reply:
It makes me happy to remove the code from that PR! Will opt for 1 :D